Benchmarks_on_OVS_integration_approaches.md

```diff
@@ -0,0 +1,213 @@
+# Benchmarks on OVS integration approaches
+
+OpenStack used to use the legacy Nova-network to provide a basic flat network model. However in its successor Neutron, this model is extended as a flat network cannot meet the requirements of a modern data center. Neutron uses a single bridge called the “integration bridge” that all guest instances are connected to provide connectivity to an external network with a physical network interface. This is essentially the same as the nova-network used to connect VMs to a physical interface.
+
+In QEMU, a L2 network interface is simulated with a tap device. However, in XenServer, vif is more frequently used because of its performance is much better. As Neutron heavily depends on tap device, it seems to be a conflict for the integration between XenServer and OpenStack. [XenAPI: OVS agent updates the wrong port with Neutron](https://review.openstack.org/#/c/242846/) introduces a new bridge called "interim bridge" to work between the guest vm and the integration bridge so vif will still be used while without invasively changing Neutron. This blog post is aiming to investigate the performance cost of "interim bridge" by setting up an environment comparing the conditions with and without interim bridge.
```

Comment:
Actually, this paragraph needs rewriting to avoid usage of terms like tap or vif that are defined in the terminology reference below.

The blog post should flow like:
* Describe the issue in a high level way
* Introduce the terminology
* Describe the issue in terms of the terminology, so we can link that to the reproduction environment
* Environment setup (as existing)
* Running tests (as existing)
* Results (as existing)